collections/Lat_misc/MS_Lat_misc_e_2.xml

```diff
@@ -37,51 +40,215 @@
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="6" xml:id="MS_Lat_misc_e_2-item6">
+                     <locus>(fols. 27r–31v) </locus>
+                     <author key="person_287984736">Hippocrates</author>
+                     <title key="work_2069">Prognostica</title>
+                     <note>(tr. <persName role="trl" key="person_305230031">Constantinus Africanus</persName> or <persName role="trl"  key="person_84814786">Gerard of Cremona</persName>)</note>
+                     <rubric>Pronostica Ypocratis</rubric>
+                     <incipit>Omnis qui medicine artis studio. seu gloriam. seu delectabilem amicorum consequi desiderat copiam </incipit>
+                     <explicit>numero dierum nostro quoque exposite ordine preceptorum</explicit>
+                     <finalRubric>Pronostica ypocratis</finalRubric>
+                     <bibl>Pr. (with commentary of Galen) in <title>Articella</title> [Padua, c. 1476], as above, fols. h1r–n7r. <title>Hippocrates Latinus</title> XLV.A.2, this ms. listed p. 202 (misdated)</bibl>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="7" xml:id="MS_Lat_misc_e_2-item7">
+                     <locus> (fols. 32r–34v) </locus>
+                     <author key="person_262260005">Philaretus</author>
+                     <title key="work_3633">Liber pulsuum</title>
+                     <rubric><add place="margin">Incipiunt P. Phylareti</add></rubric>
+                     <incipit>Intencionem habemus in presenti conscripcione de pulsuum negocio compendiosam exponere tradicionem</incipit>
+                     <explicit>defectionem et mortem significat</explicit>
+                     <bibl>Cf. P. Morpurgo, ‘Il commento al de pulsibus Philareti di Mauro Salernitano: Introduzione ed edizione critica dal ms. Parisinus Latinus 18499’, <title>Dynamis: Acta Hispanica ad Medicinae Scientiarumque Historiam Illustrandam</title>, 7–8 (1987–88), 307–46, from 317; using the present MS (siglum ‘L’); among the six MSS. used for comparison with the Paris MS., four others are also at the Bodleian: MS. Ashmole 399, fols. 62r–63r; MS. Ashmole 1285, fols. 181r–182v; MS. Auct. F. 5. 30, fols. 27v–29v; and MS. Laud Lat. 65, fols. 61r–64v.</bibl>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  <msItem n="8" xml:id="MS_Lat_misc_e_2-item8">
+                     <locus>(fols. 35r–54r) </locus>
+                     <author key="person_44299175">Galen</author>
+                     <title key="work_1621">Tegni</title>
+                     <rubric><add place="margin">Tengni G. phylosofi </add></rubric>
+                     <rubric><damage>    </damage> phylosofi</rubric>
```

Comment:
Is the number of spaces meant to indicate the number of missing characters? If so, would `<gap reason="illegible" quantity="3" unit="char"/>` (or `reason="lost"`) be better?

Reply:
It was supposed to represent an unspecified amount of damage, but a better solution would just be to have a defective attribute on `rubric`, so I'll change to that.